### PR TITLE
sidebar icons fixed

### PR DESCRIPTION
**Pull request by shelar1423**
Fixes issue : #246 
image
image

### CODE REVIEW

File: quickstarts/kotlin.mdx [hunk 1]
```diff
@@ -84,33 +84,42 @@ The workflow includes the following:
...
+  Proper authorization needs to be set for the Chat channel for subscribers.
 </Note>
 
 Please proceed to create a workflow.
 
 1. Click **Workflows** on the left sidebar of your Novu dashboard.
 2. Click the **Create Workflow** button on the top right.
-<Frame caption="Use the create workflow button to create a workflow"><img src="/images/workflow-kotlin.png" /> </Frame>
+   <Frame caption="Use the create workflow button to create a workflow">
+     <img src="/images/workflow-kotlin.png" />{" "}
```
Comment (unicodeveloper) [inline quickstarts/kotlin.mdx:104]: What does `{" "}` do and why did you add it?

File: quickstarts/nextjs.mdx [hunk 2]
```diff
@@ -219,33 +238,35 @@ export default async function createTopic(req, res) {
...
 ## Add subscriber to a Topic
 
 The code for adding a subscriber to a previously created topic is as follows:
 
-
 <Note>
-You can only add those subscribers to a topic that you've already created. You can see all the subscribers in the [Novu web dashboard](https://web.novu.co/subscribers)
+  You can only add those subscribers to a topic that you've already created. You
+  can see all the subscribers in the [Novu web
+  dashboard](https://web.novu.co/subscribers)
```
Comment (unicodeveloper) [inline quickstarts/nextjs.mdx:260]: Why are you formatting it this way?

**Comment by unicodeveloper** [comment]
They look fine. Please attend to the comments @shelar1423

**Comment by shelar1423** [comment]
@unicodeveloper sure  I'll resolve them and update you,ig it the idle auto format issue I'll fix it

**Comment by shelar1423** [comment]
shall i recode  ? @unicodeveloper

**Comment by shelar1423** [comment]
@unicodeveloper raising another pr closing this one
